src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -882,17 +831,23 @@ export default class ConvasController {
         const alerts = aircraft.hasAlerts();
 
         if (!aircraft.inside_ctr) {
-            cc.fillStyle = COLORS.LIGHT_SILVER_03;
```

Comment:
I think these are/were for drawing a different color dot (aircraft) based on the various states.  I don't know if any of those are working, though.